tasks/main.yml

```diff
@@ -37,6 +37,7 @@
   dnf:
     name: "@Infiniband Support"
     state: present
+    nobest: true
```

Comment:
Are you using a Pulp mirror of the OFED repo? Currently pulp-rpm is broken for CentOS 8 based mirrors, leading to broken deps. I only mention it because --nobest sometimes helped to install things, but wasn't really fixing the issue.

Reply:
I think I've reverted this, I was trying to install OFED on CentOS Stream and no, don't do that.